app/components/header/free-weeks-left-button.hbs

```diff
@@ -1,4 +1,9 @@
-<PrimaryLinkButton @size="extra-small" @route="pay" class="mr-4" data-test-free-weeks-left-button>
+<PrimaryLinkButton
+  @size={{this.linkButtonSize}}
+  @route="pay"
+  class="{{if (eq this.linkButtonSize 'extra-small') 'mr-3'}}"
```

Comment:
Same note as elsewhere - we shouldn't have to set margins inside components and should instead have the parent pass them in. 

The principle to follow here is: a component is roughly a "box", and it should ideally only control everything inside the box, and nothing outside the box. Margins affect outer layout, which is the responsibility of the parent component / template.